srcpkgs/python3-yapf/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'python3-yapf'
+pkgname=python3-yapf
+version=0.29.0
+revision=1
+archs=noarch
+create_wrksrc=yes
+build_wrksrc="yapf-${version}"
```

Comment:
Shouldn't `wrksrc="yapf-${version}"` work instead of having `create_wrksrc` and `build_wrksrc`?

Reply:
Yes it should work, much better in this case.